downstream/modules/platform/ref-controller-data-collection-details.adoc

```diff
@@ -0,0 +1,43 @@
+[id="ref-controller-data-collection-details"]
+
+= Details of data collection
+
+{Analytics} collects the following classes of data from {ControllerName}:
+
+* Basic configuration, such as which features are enabled, and what operating system is being used
+* Topology and status of the {ControllerName} environment and hosts, including capacity and health
+* Counts of automation resources:
+
+** organizations, teams, and users
+** inventories and hosts
+** credentials (indexed by type)
+** projects (indexed by type)
+** templates
+** schedules
+** active sessions
+** running and pending jobs
+* Job execution details (start time, finish time, launch type, and success)
+* Automation task details (success, host id, playbook/role, task name, and module used)
+
+You can use `awx-manage gather_analytics` (without `--ship`) to inspect the data that {ControllerName} sends, so that you can satisfy your data collection concerns. 
+This creates a tarball that contains the analytics data that would be sent to Red Hat.
```

Comment:
Suggested re-write:

"This creates a tarball that contains the analytics data that is sent to Red Hat."